.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,41 @@
+name: 'coverage-report'
+on:
+  push:
+    branches:
+      - main
+      - chore/code-coverage # remove before merge to main
+
+permissions:
+  contents: write
+jobs:
+  coverage-report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          coverage: none
+          tools: composer:v2
+          extensions: xdebug
+      - name: Install Dependencies
+        run: composer install -q --profile --ignore-platform-reqs --no-interaction --no-ansi --no-scripts --no-suggest --prefer-dist
+      - uses: php-actions/phpunit@v3
```

Comment:
Please, use commit SHA for 3rd party actions

Reply:
Instead I'm using local docker-compose command